src/components/pages/home.jsx

```diff
@@ -32,28 +34,32 @@ const HomePage = React.createClass({
     return {inputValue: ""}
   },
   handleClearSearchClicked() {
-    this.setState({inputValue: ""})
-    this.context.router.push("")
+    const query = ""
+    this.setState({inputValue: query})
+    this.context.setQuery(query)
+    this.context.router.push(query)
```

Comment:
I'm not sure on how to do it. There are many different variations 🤔 

Please come and see me if you'd like to.

I added a comment about `scrollIntoView` in my next commits